packages/create-canon/scaffold/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "canon-app",
+  "description": "A new instance of a canon app.",
+  "version": "1.0.0",
+  "private": true,
+  "main": "./index.js",
+  "scripts": {
+    "dev": "canon-dev",
+    "build": "canon-build",
```

Comment:
We found most sites need to also increase the `max-old-space-size` of node before building, do you have a strong preference on removing that from the default build script? Here's the old version:
`"build": NODE_OPTIONS=--max-old-space-size=4096 canon-build`

Reply:
The problem with that command is that only works on POSIX, but neither CMD nor PowerShell accept inline environment variables on commands. We would need to use the `cross-env` package to set it.
But in any case, I'd look into what causes the building process to use so much memory instead of this, because there might be a case where the user tries to run it in a computer without that much memory anyway.